src/Sarif/DefaultTraces.cs

```diff
@@ -10,7 +10,8 @@ public enum DefaultTraces
     {
         None,
         ScanTime = 0x01,
-        ScanExecution = 0x2,
+        ScanExecution = 0x02,
         RuleScanTime = 0x04,
+        MemoryUsage = 0x08,
```

Comment:
>MemoryUsage [](http://example.com/codeflow?start=8&length=11)

Should we have a global memory usage vs. peak mem data point for the end-to-end scan? And the one per rule/per target? Who is asking for this feature, by the way?

Reply:
fixed to only log once per end of the PR review.